aten/src/ATen/cuda/CublasHandlePool.cpp

```diff
@@ -48,6 +48,41 @@ void destroyCublasLtHandle(cublasLtHandle_t handle) {
 }
 
 using CuBlasLtPoolType = DeviceThreadHandlePool<cublasLtHandle_t, createCublasLtHandle, destroyCublasLtHandle>;
+
+// ugly hack until hipblasSetWorkspace exists
+#include <rocblas/rocblas.h>
+
+static hipblasStatus_t rocBLASStatusToHIPStatus(rocblas_status error)
+{
+    switch(error)
+    {
```

Comment:
Nit (not sure why clang-format is not covering those)
```suggestion
static hipblasStatus_t rocBLASStatusToHIPStatus(rocblas_status error) {
    switch(error) {
```